app/helpers/calculate-user-score.js

```diff
@@ -0,0 +1,5 @@
+module.exports.calculateUserScore = (newScore, currentScore) => {
+  if (currentScore != 0)
+    return newScore === currentScore ? newScore * -1 : newScore * 2;
```

Comment:
Why this? can you explain?

Reply:
It is for these requirements:
![image](https://user-images.githubusercontent.com/31294096/143496197-17a59c42-49ba-4bc7-b726-09f5eba2b378.png)

- If you want to set the same value then you have to remove the current adding the opposite value. `newScore * -1`.
- else you have to remove the old value and add the new one. `newScore * 2`.

Reply:
but why are you multiplying?

Reply:
To set the sign and add or substract in the service. If the `currentScore =1` and the `newScore=1`( are the same value ) then i need substract 1 to user score `newScore * -1`  ` 1 * -1 =  -1`.  then `newUserScore =-1`. in the  service substract 1: 
`await user.update({ score: (newUserScore + currentUserScore) }, { transaction });`
the same when newScore is negative.
I think I could improve the variables names to be clear.